### PR TITLE
Release notes 4.3 updates

### PR DESCRIPTION
**Pull request by codyhoag**
Updates from first half of *What's New* presentation.

### CODE REVIEW

File: release_notes/ocp-4-3-release-notes.adoc [hunk 1]
```diff
@@ -112,8 +112,10 @@ and xref:../installing/installing_gcp/installing-gcp-private.adoc#installing-gcp
 ==== Automatic rotation of certificates
 
 Automated service CA rotation will be available in this release in a future
-z-stream update. This is to allow time for administrators to plan accordingly
-for their environments.
+z-stream update. In previous versions, the service CA did not automatically
+renew, leading to service disruption and requiring manual intervention. With the
+service CA and signing key auto-rotating before expiration, administrators can
```
Comment (ahardin-rh) [inline release_notes/ocp-4-3-release-notes.adoc]: Consider rewording as:
The service CA and signing key auto-rotates before expiration. This allows administrators to plan for their environments in advance, avoiding service disruption.

File: release_notes/ocp-4-3-release-notes.adoc [hunk 2]
```diff
@@ -353,14 +424,35 @@ changes.
...
+* Support for Prometheus metrics.
+* Usage of Red Hat Universal Base Image (UBI).
+* Molecule-based end to end testing.
+
+Lastly, the Golang-based Operator improvements include
+
+* OpenAPI spec generation.
+* Kubernetes 1.14 support.
+* Removal of `dep`-based projects. All Go projects are now scaffolded to use Go
+modules. The `operator-sdk new` command's `--dep-manager` flag has likewise been
```
Comment (ahardin-rh) [inline release_notes/ocp-4-3-release-notes.adoc]: s/has likewise been/has been removed

**Comment by ahardin-rh** [comment]
@codyhoag Just some minor suggestions. Otherwise, looks great!

**Comment by codyhoag** [comment]
Thanks, @ahardin-rh!

**Pull request merged**